packages/cli/test/fixtures/discover/mem.datasource.js.txt

```diff
@@ -0,0 +1,106 @@
+const DataSource = require('loopback-datasource-juggler').DataSource;
```

Comment:
I would prefer to avoid direct dependency on loopback-datasource-juggler and import `DataSource` class constructor from `@loopback/repository`.

Not a big deal, we can improve this in a follow-up pull request.